### PR TITLE
Feature/teamdisplay

### PR DESCRIPTION
**Pull request by Jesuitman**
# Description

This allows you to add to your team and update it, there is currently a bug where it only partially works, tackling that next

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>PokéLex</title>
```
Comment (tenthwalker) [inline public/index.html:27]: way to implement prior project feedback!

File: src/App/App.js [hunk 2]
```diff
@@ -14,19 +16,52 @@ function App() {
...
+    const updatedTeam = [...team];
+    updatedTeam[index] = null;
+    setTeam(updatedTeam);
+  };
+
   return (
     <div className="App">
       <header className="App-header">
-        {/* Your existing header content */}
+        <h1>PokéLex</h1>
```
Comment (nicktu12) [inline src/App/App.js:42]: Dont forget to implement routing - I really wanna see the polish on this showcase app!

File: src/TeamDisplay/TeamDisplay.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    text-align: center;
+  }
+  
+  .team-section h2 {
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+  
+  .team-grid {
```
Comment (tenthwalker) [inline src/TeamDisplay/TeamDisplay.css:12]: I know we talked about flex vs grid, let me know if you have any questions while looking at grid implementation

File: public/index.html (hunk 1 above)
Comment (nicktu12) [inline public/index.html:27]: Nice touch!! There are some other meta properties in the head that may be worth updating!

File: src/App/App.js [hunk 4]
```diff
@@ -2,9 +2,11 @@ import React, { useState } from 'react';
 import './App.css';
 import List from '../List/List';
 import PokemonCard from '../PokemonCard/PokemonCard';
+import TeamDisplay from '../TeamDisplay/TeamDisplay'; // Import the TeamDisplay component
 
 function App() {
   const [selectedPokemon, setSelectedPokemon] = useState(null);
+  const [team, setTeam] = useState(Array(6).fill(null));
```
Comment (nicktu12) [inline src/App/App.js:9]: Array.fill - nice!!

**Review by tenthwalker — APPROVED** [review]
This is coming together really nicely, just remember to make more atomic commits!
I especially appreciate the styling that mimics the brand styles and how clean/blocky the display is for ease of use by younger/limited motor skills users or for potential mobile device access.

**Pull request merged**